src/editortoolkit_cmn.cpp

```diff
@@ -247,15 +246,15 @@ bool EditorToolkitCMN::KeyDown(std::string &elementId, int key, bool shiftKey, b
 
     // For elements whose y-position corresponds to a certain pitch
     if (element->HasInterface(INTERFACE_PITCH)) {
-        PitchInterface *interface = element->GetPitchInterface();
-        assert(interface);
```

Comment:
You should not have to change this. Please revert

Reply:
I will revert that change.  Though I am quite frustrated that the only time I get a directed response from you is telling me I did something wrong, when I've asked you how to do things in this library in 3 different ways over the last week.

Reply:
Sorry about that. This is a part of the code that hasn't been touched for years. One reason is that improving it needs proper thinking about how to refactor it. Guiding you would require quite some time that I do not have right at the moment. I understand you would expect clear and elaborated answers straight away but I am sorry this is not always possible. Nonetheless, the PR looks fine and I am happy to merge it.